prefix_tags/prefix_tags_config.c

```diff
@@ -78,9 +43,39 @@ int tags_parse_ip_prefix(const char *ip_prefix, ip_addr_t *addr, uint32_t *prefi
    return 0;
 }
 
+/** Dealloc ipps_network_list_t
+ * Dealloc struct ipps_network_list_t, opposite of load_network
```

Comment:
> ... opposite of load_network

I don't see `load_network` anywhere.

Reply:
Original author didn't include load_networks and destroy_networks into API. It is probably because of the `data` pointer, which might point to some allocated structure...
We will think about putting at least destroy_networks into API in the future.